libagent/device/ledger.py

```diff
@@ -72,7 +72,7 @@ def pubkey(self, identity, ecdh=False):
     def sign(self, identity, blob):
         """Sign given blob and return the signature (as bytes)."""
```

Comment:
```suggestion
        """Sign given blob and return the signature (as bytes)."""
        """Note (mc): In case that the blob is 32 bytes, we ignore ssh protocol marker in the identity. This allows to produce a direct ed25519 signature on a 32 byte blob."""
```